.github/workflows/pull_request_approved.yml

```
@@ -2,7 +2,7 @@ name: Notify about approved but still open PRs
 
 on:
   schedule:
-    - cron: '0 0 * * *' # Once a day
+    - cron: '0 13 * * 1' # Every Monday at 1PM UTC (9AM EST)
```

```suggestion
    - cron: '0 13 2-30/2 * *' # Every even day of the month at 1PM UTC (9AM EST)
```
How about every other day?

---

Probably even better than this one I had `0 13 */2 * *`